templates/circuit.yaml

```diff
@@ -158,3 +158,28 @@ spec:
             path: /app/circuit_layers.json
           - name: circuit-connectivity
             path: /app/circuit_connectivity.json
+
+    - name: create-random-circuit
+      parent: generic-task
+      inputs:
+        parameters:
+          - name: num-qubits
+          - name: num-gates
+          - name: seed
+            default: "None"
+          - name: command
+            value: python3 main_script.py
+        artifacts:
+          - name: main-script
+            path: /app/main_script.py
+            raw:
+              data: |
+                from zquantum.core.circuit import save_circuit
+                from zquantum.core.testing import create_random_circuit
```

Comment:
Since it seems we'll start using this function outside of testing, perhaps it would make sense to move it from `testing` to `utils`?

Reply:
Even though this function is now used inside a template, it will be used primarily for testing other templates.